backend/services/seeder.py

```
@@ -326,7 +327,7 @@ def seed_database(db: Session):
 
             if status == "Finished" and m.get("home_score") is not None and m.get("away_score") is not None:
                 try:
-                    settle_result(fixture, int(m["home_score"]), int(m["away_score"]))
+                    finish_fixture(fixture, int(m["home_score"]), int(m["away_score"]), db, update_standings=False)
                 except Exception:
                     pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)